sourced/ml/__main__.py

```diff
@@ -10,12 +10,11 @@
     run_swivel, postprocess_id2vec, preprocess_id2vec, repos2coocc_entry, repos2df_entry, \
     repos2ids_entry, repos2bow_entry, repos2roles_and_ids_entry, repos2id_distance_entry, \
     repos2id_sequence_entry
-from sourced.ml.cmd_entries.args import add_repo2_args, add_feature_args, \
-    add_vocabulary_size_arg, add_extractor_args, add_split_stem_arg, \
-    ArgumentDefaultsHelpFormatterNoNone
-from sourced.ml.cmd_entries.repos2bow import add_bow_args
+from sourced.ml.cmd_entries.args import add_df_args, add_feature_args, \
+    add_vocabulary_size_arg, add_repo2_args, add_split_stem_arg, \
+    ArgumentDefaultsHelpFormatterNoNone, add_bow_args
 from sourced.ml.cmd_entries.run_swivel import mirror_tf_args
-from sourced.ml.utils import install_bigartm, add_engine_args
```

Comment:
Also be carefull with apollo. https://github.com/src-d/apollo/blob/master/apollo/__main__.py#L10

Reply:
yes no worry, i updated my PR : https://github.com/src-d/apollo/pull/39